src/Updater/UpdateContext.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phpcq\Runner\Updater;
+
+use Phpcq\GnuPG\Signature\SignatureVerifier;
+use Phpcq\Runner\Downloader\DownloaderInterface;
+use Phpcq\Runner\Repository\InstalledRepository;
+use Phpcq\Runner\Composer;
+use Symfony\Component\Filesystem\Filesystem;
+
+/** @psalm-immutable  */
+final class UpdateContext
```

Comment:
Can we bump to PHP 7.4 and use typed properties now?
When having public properties, I'd prefer this.